tests/behat/features/accessibility.feature

```diff
@@ -65,3 +65,100 @@ Feature: Accessibility
     And the ".cms-container-skip-link-target" element should have focus
     When I press the "Tab" key globally
     And the ".ui-tabs-tab[aria-controls='Root_Main']" element should have focus
+
+  Scenario: Site tree keyboard navigation
+    Given a "Page" "Page 1"
+    And a "Page" "Page 2"
+    And the "Page" "Page 2a" is a child of a "Page" "Page 2"
+    And the "Page" "Page 2b" is a child of a "Page" "Page 2"
+    And the "Page" "Page 2bi" is a child of a "Page" "Page 2b"
+    And the "Page" "Page 2bii" is a child of a "Page" "Page 2b"
+    And the "Page" "Page 2biii" is a child of a "Page" "Page 2b"
+    And the "Page" "Page 2c" is a child of a "Page" "Page 2"
+    And a "Page" "Page 3"
+    And I am logged in with "ADMIN" permissions
+    And I go to "/admin/pages"
+
+    # Move focus to the element before the site tree so we have a known starting point
+    When I focus on the ".font-icon-list" element
```

Comment:
Don't use `.font-icon-*` css class as a behat selector for interactive elements, since https://github.com/silverstripe/silverstripe-admin/issues/1957 is moving these into child elements instead.

Reply:
Done